.nvmrc

```diff
@@ -0,0 +1 @@
+v18.19.1
```

Comment:
Shouldn't we prefer setting only the major version in order to ensure security updates are applied whenever possible?
```suggestion
18
```